releasenotes/notes/process-agent-status-command-6ea4385bea90836f.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Added new `process-agent status` command. This command has extra debug info that cannot be found in `process-agent --info`
```

Just a small suggestion to make the intention clear.
```suggestion
    Added new `process-agent status` command to help with troubleshooting and for better consistency with the core Agent. This command is intended to eventually replace `process-agent --info`.
```

---

Oh yeah this is much better, thanks for the suggestion :)